homeassistant/components/manual/alarm_control_panel.py

```diff
@@ -32,15 +32,16 @@
     STATE_ALARM_TRIGGERED,
 )
 from homeassistant.core import HomeAssistant, callback
-from homeassistant.exceptions import HomeAssistantError
+from homeassistant.exceptions import ServiceValidationError
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.event import async_track_point_in_time
 from homeassistant.helpers.restore_state import RestoreEntity
+from homeassistant.helpers.template import Template
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 import homeassistant.util.dt as dt_util
 
-_LOGGER = logging.getLogger(__name__)
+DOMAIN = "manual"
```

Comment:
Maybe move DOMAIN, CONF etc. to a `const.py`?

Reply:
Constants should only be in a const.py module if the constants are used in more than one module.

Reply:
Good to know, thanks!